migrations/20200516225721-initial-table-creation.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
```

Comment:
Well that explains the routes erroring out. You never create the genres table or the linking table for connecting genres and novels.